lib/data_update_scripts/20220126102900_create_home_navigation_link.rb

```diff
@@ -0,0 +1,20 @@
+module DataUpdateScripts
+  class CreateHomeNavigationLink
+    def run
+      NavigationLink.create_or_update_by_identity(
+        name: "Home",
+        url: URL.url("/"),
+        icon: File.read(Rails.root.join("app/assets/images/twemoji/house.svg")),
+        display_only_when_signed_in: false,
+        position: 1,
+        section: :default,
+      )
+
+      # Increment all other default section links to account for new home link, and update to be 1-based
```

Comment:
It is possible that if an admin has already adjusted their navigation links to be 1-based, they'll now see:

```
1. Home
3. The link that used to be at position 1
```

There shouldn't be any negative effect on the actual order of the links in the UI, so I'm tempted to think this is fine, and less strange than seeing "Home" appear at position -1

Reply:
Definitely a good catch, I suppose we could test "what's the beginning position" and use that to inform our update.  But not worth a "blocker"

Reply:
What if they didn't only change the beginning number but reordered them? Won't we override their order? IMO this could be a potential blocker as I'd feel bad about us undoing something an admin explicitly did.